projects/rero/ng-core/src/lib/record/detail/detail.component.html

```diff
@@ -22,11 +22,11 @@
     <div class="col-3" *ngIf="record">
         <a href="javascript:void(0);" class="btn btn-sm btn-secondary" routerLink="../../edit/{{ record.metadata.pid }}">
             <i class="fa fa-pencil"></i>
-            {{ "Edit"|translate }}
+            {{ 'Edit'|translate }}
         </a>
         <a href="javascript:void(0);" class="btn btn-sm btn-secondary ml-1">
             <i class="fa fa-trash"></i>
-            {{ "Delete"|translate }}
+            {{ 'Delete'|translate }}
```

Comment:
For consistency, should we put spaces arround "|" ?

Reply:
done